### ISSUE TITLE
interface and type style

### ISSUE THREAD

**Issue by TrySound** [nzakas/eslint-plugin-typescript#17]
`interface` and `type` definitions accepts both and comma and semicolon separators

```
interface IName {
  prop1: number,
  prop2: string
}
```
```
interface IName {
  prop1: number;
  prop2: string;
}
```

We need to add a rule for this to make code more consistent.

**Comment by weirdpattern** [comment]
Not sure if this has to be triaged or not, but I would like to give it a try if nobody is working on this.

**Comment by weirdpattern** [comment]
Interesting, seems @JamesHenry discussed this with the TypeScript team a few months back...
[here](https://github.com/Microsoft/TypeScript/issues/10730#issuecomment-245443010)

Assuming this is still needed, correct @JamesHenry?

**Comment by JamesHenry** [comment]
Sure, a rule for this would be great! Thanks guys

**Comment by weirdpattern** [comment]
alright, I already have some work done, I will create the PR this weekend

**Comment by weirdpattern** [comment]
ok, this time is for real... I have some code, just need to complete the documentation on this rule.
Will do this if not tonight, tomorrow night

**Comment by weirdpattern** [comment]
PR #48 has been submitted

**Comment by weirdpattern** [comment]
@JamesHenry this can be closed now that #48 has been merged ☺️
